proto/process/agent.proto

```diff
@@ -465,3 +474,36 @@ message Host {
 message DNSEntry {
   repeated string names = 1;
 }
+
+// reference https://github.com/kubernetes/kubernetes/blob/cb19b56831d54d1d31249949318ef0b07bf00df9/staging/src/k8s.io/api/core/v1/generated.proto
+message Pod {
+	string name = 1;
+	string namespace = 2;
+	int64 creationTimestamp = 3;
+	string IP = 4;
+	string nominatedNodeName = 5;
+	string nodeName = 6;
+	string phase = 7;
+	int32 restartCount = 8;
+	repeated ContainerStatus containerStatuses = 9;
+	repeated string labels = 10;
+	repeated string annotations = 11;
+	repeated OwnerReference ownerReferences = 12;
+	repeated string tags = 13;
+	bytes yaml = 14;
+}
+
+message ContainerStatus {
+	string name = 1;
+	string containerID = 2;
+	bool ready = 3;
+	int32 restartCount = 4;
```

Comment:
Just want to make sure that container could also have an independent restart compare to pod?

Reply:
yeah in fact the restart on the pod is the sum of the restarts of the containers, maybe we should just keep the breakdown and compute the sum on display? Or should we duplicated like I did the info at the pod level?

Reply:
I prefer to duplicate. This would avoid having to query for all containers and do the sum if only a number is needed. Thanks for the explanation.

Reply:
Oh this count is only in `ContainerStatus` I thought it was in container objects. Hmm, then I don't feel strong of duplicating it.